.github/workflows/enteprise.yml

```diff
@@ -9,7 +9,7 @@ on:
       - '*.md'
 
 env:
-  DTAS_VERSION: v1.1.2
+  DTAS_VERSION: OPSEXP-1645-acctests
```

Comment:
We should probably release something here.
Generally speaking I think we could release a version at the end of each sprint, so we can use it in the deployment tools releases beginning of the next sprint.
If that works for you I'll add a release task for DTAS 1.2.0